kpconv_torch/utils/tester.py

```diff
@@ -389,7 +390,11 @@ def load_output_generators(
                                     np.argmax(probs, axis=1)
                                 ].astype(np.int8)
                                 self.preds_on_all_files += [preds]
-                                yield point_batch, preds, probs
+                                # Deduce the corresponding RGB triplets
+                                color_palette = colors.convert_hex_to_rgb(
+                                    test_loader.dataset.colors
+                                )
```

Comment:
`config["colors"]`

Colors are not loaded in the dataset, they are left in the config dictionary.